hyperspy/misc/io/tools.py

```diff
@@ -76,6 +77,33 @@ def ensure_directory(path):
     if directory and not os.path.exists(directory):
         os.makedirs(directory)
 
+def find_filenames_matching_substrings(path,substrings,sortfiles=True):
```

Comment:
This is not used anywhere and not tested.

Reply:
removed